### PR TITLE
Move markers to more appropriate places

### PR DESCRIPTION
**Pull request by markusthoemmes**
This should get rid of marker confusion when running sequences. @RSulzmann please confirm.

### CODE REVIEW

**Comment by rabbah** [comment]
what's the goal for this marker? measure total time for an activation (so for a sequence, the totality of all activations within)?

**Comment by RSulzmann** [comment]
Calculating performance data for sequence actions has two problems:

1. The start and finish of markers CONTROLLER_ACTIVATION (CONTROLLER_ACTIVATION_BLOCKING) were not in sync

The above changes should fix this.

2. Because all activations inside a sequence have the same Tid, the perftool cannot relate the different marker entries to a single activation and calculate correct durations.

do we plan to adress this also ?

File: core/controller/src/main/scala/whisk/core/controller/actions/PrimitiveActions.scala [hunk 1]
```diff
@@ -152,19 +150,12 @@ protected[actions] trait PrimitiveActions {
...
-            case Failure(t) =>
-                // short circuits db polling
-                promise.tryFailure(t)
-        }
-
-        response // will either complete with activation or fail with timeout
+        // Will either complete with activation or timeout
+        promise.future
+            .withTimeout(totalWaitTime, new BlockingInvokeTimeout(activationId))
+            .andThen {
```
Comment (rabbah) [inline core/controller/src/main/scala/whisk/core/controller/actions/PrimitiveActions.scala]: this could just be `recover` no?

Comment (markusthoemmes) [inline core/controller/src/main/scala/whisk/core/controller/actions/PrimitiveActions.scala]: Nope. `tryFailure` returns a `Boolean` which'll alter the return-type. `andThen` just passes the original future through and only applies the sideeffect.

**Comment by markusthoemmes** [comment]
@rabbah any more concerns?

**Comment by rabbah** [comment]
I approved the changes.

**Pull request merged**